benchmarks/src/garage_benchmarks/experiments/algos/ddpg_garage_tf.py

```diff
@@ -5,7 +5,7 @@
 from garage import wrap_experiment
 from garage.envs import normalize
 from garage.experiment import deterministic
-from garage.np.exploration_strategies import OUStrategy
+from garage.np.exploration_policies import OUPolicy
```

Comment:
Hmm, at the cost of being too verbose, should we name these OUExplorationPolicy, GaussianExplorationPolicy, etc?

Rationale: If you come across a random snippet of code, it's not clear that these are wrapping the main policy.

We could also use verb named, e.g. AddOUNoisePolicy AddGaussianNoisePolicy, etc.

Reply:
I actually think the verb form without Policy at the end works best. What do you think?

Reply:
I agree